src/configs/typescript.ts

```diff
@@ -36,6 +36,12 @@ export function typescript(
     'ts/unbound-method': 'error',
   }
 
+  const tsconfigPath = typeof tsc === 'string' && tsc.trim()
```

Comment:
Why `trim` is needed?

Reply:
empty string check

Reply:
```suggestion
  const tsconfigPath = typeof tsc === 'string' ? (
     tsc.trim() !== '' ? [tsc] : null
   ) : ...
```

Reply:
I pushed a change. I think that setting `tsconfigPath: " "` should be a user error, and we shouldn't tolerant that implicitly.

Reply:
agree, thanks
_just a habit to deal with user input - but here we have 'advanced' users_